modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/deprecated-annotation.yaml

```diff
@@ -0,0 +1,15 @@
+- name: kubernetes.extended-monitoring.deprecated-annotation
+  rules:
+  - alert: ExtendedMonitoringDeprecatatedAnnotation
+    expr: >-
+      group by (kind,namespace,name) (d8_deprecated_legacy_annotation == 1)
+    labels:
+      severity_level: "4"
+    annotations:
+      plk_protocol_version: "1"
+      plk_markup_format: "markdown"
+      plk_create_group_if_not_exists__d8_extended_monitoring_deprecated_annotation: "D8ExtendedMonitoringDeprecatedAnnotation,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
+      plk_grouped_by__d8_extended_monitoring_deprecated_annotation: "D8ExtendedMonitoringDeprecatedAnnotation,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
+      summary: >
+        here is a deprecated `extended-monitoring.flant.com/enabled` annotation on {{ $labels.kind }} {{ $labels.namespace }}/{{ $labels.name }}.
+        Migrate to `extended-monitoring.deckhouse.io/enabled` label ASAP.
```

Comment:
In this alert, i suggest providing example commands. Since we actually know kind, namespace and name in this alert, we can add particular shell commands

- how to find objects with legacy annotation
- how to remove the annotation
- how to add the new one

Or even we should provide oneliner for that stuff.

Reply:
That's up to the user. We won't describe every single way to change objects in the Kubernetes API. These Namespaces and objects may be deployed from git or reconciled via some operator. No way to know all of them.